tests/bench/oolong.py

```
@@ -0,0 +1,600 @@
+                actual_response=actual_response,
+                score=score,
+                context_length_tokens=context_length,
+            )
+            result["score"] = score
+            result["passed"] = score >= 0.99
+            result["output_lines"] = [
+                f"Question: {question}",
+                f"Expected: {gold_answer}",
+                f"Score: {score:.3f}",
+            ]
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)